modules/core/MapSystem.js

```diff
@@ -682,24 +678,26 @@ export class MapSystem extends AbstractSystem {
   }
 
 
-  // convenience methods for zomming in and out
-  _zoomIn(delta)  { return this.centerZoom(this.center(), ~~this.zoom() + delta, 250); }
-  _zoomOut(delta) { return this.centerZoom(this.center(), ~~this.zoom() - delta, 250); }
+  // convenience methods for zooming in and out
+  _zoomIn(delta)  { return this.setMapParams(undefined, ~~this.zoom() + delta, undefined, 250); }
```

Comment:
yea this is true - I made it so that everything goes through `setMapParams` eventually.   But we could just call `this.zoom` here to avoid all the `undefined` args.

`setMapParams` is the replacement for `setCenterZoom` (which we called _everywhere_), but now it gained another parameter to set rotations too.